src/app/lib/config/services/persian.js

```diff
@@ -58,11 +58,11 @@ const persian = {
             'متاسفانه صفحه مورد نظر شما را پیدا نمی‌کنیم. لطفا گزینه‌های زیر را امتحان کنید:',
           solutions: [
             'آدرس صفحه مورد نظر را دوباره بررسی کنید',
-            'با مرورگر خود صفحه را ریفرش (refresh) کنید',
+            'با مرورگر خود صفحه را ریفرش (Refresh) کنید',
             'در نوار جستجوی بی‌بی‌سی دنبال اطلاعات مورد نظرتان بگردید',
           ],
           callToActionFirst: 'برای یافتن اطلاعات مورد نظر به ',
```

Comment:
```suggestion
          callToActionFirst: 'برای یافتن اطلاعات مورد نظر به صفحه',
```

Reply:
This is a change @sareh requested above because the phrase "Home Page" is being split across diffs.